configuration/appframework/scripts/isPatientEnrolledInProgramOfUuid.js

```diff
@@ -0,0 +1,6 @@
+function isPatientEnrolledInProgramOfUuid(programUuid) {
```

Comment:
Let's name this more concisely - we don't need `InProgramOfUuid` in the file name or method name